addons/im_livechat/controllers/chatbot.py

```diff
@@ -63,22 +62,33 @@ def chatbot_trigger_step(self, channel_id, chatbot_script_id=None):
             return None
 
         posted_message = next_step._process_step(discuss_channel)
-        return {
-            "data": Store(posted_message, for_current_user=True).get_result(),
-            'scriptStep': {
-                'id': next_step.id,
-                'answers': [{
-                    'id': answer.id,
-                    'label': answer.name,
-                    'redirectLink': answer.redirect_link,
-                } for answer in next_step.answer_ids],
-                'isLast': next_step._is_last_step(discuss_channel),
-                'message': plaintext2html(next_step.message) if not is_html_empty(next_step.message) else False,
-                'type': next_step.step_type,
-            },
-            'operatorFound': next_step.step_type == 'forward_operator' and len(
-                discuss_channel.channel_member_ids) > 2,
+        store = Store()
+        store.add(posted_message, for_current_user=True)
+        store.add(next_step)
+        step_basic_info = {
+            "id": (next_step.id, discuss_channel.id),
+            "scriptStep": Store.one_id(next_step),
+            "message": Store.one_id(posted_message),
         }
+        store.add(
+            "ChatbotStep",
+            {
+                **step_basic_info,
+                "isLast": next_step._is_last_step(discuss_channel),
+                "operatorFound": next_step.step_type == "forward_operator"
+                and len(discuss_channel.channel_member_ids) > 2,
+            },
+        )
+        store.add(
+            "Chatbot",
+            {
+                "id": (chatbot.id, discuss_channel.id, "discuss.channel"),
+                "script": store.one_id(chatbot),
+                "thread": store.one_id(discuss_channel),
```

Comment:
I want to remove `one_id`, `one` + `only_id=True` should be used if no data should be sent.